scripts/build_plugins.sh

```diff
@@ -51,12 +51,21 @@ else
 fi
 p=${BUILD_JOBS:-"${p}"}
 
+# disable parallel xargs where not supported (ie on busybox)
+if [[ ! -x "$(command -v readlink)" || "$(readlink $(which xargs))" == "/bin/busybox" ]]; then
```

Comment:
OS X doesn't support that -f flag, and I was trying to keep the check simple by not having additional layers of conditionals based on OS.

What I can do is check for busybox only in the Linux case, not the Darwin case. That's probably a safe assumption (have you ever seen a Darwin with busybox ?)